src/screens/UserManagement/UserRoleEntry.res

```diff
@@ -110,38 +135,66 @@ let make = () => {
     {
       title: "Roles",
       renderContent: () =>
-        <DefaultLandingPage
-          title="Coming Soon" height="60vh" overriddingStylesTitle="text-3xl font-semibold"
-        />,
+        <div className="mt-5">
```

Comment:
<img width="660" alt="image" src="https://github.com/juspay/hyperswitch-control-center/assets/120017870/bde500ee-5843-4af4-a8a0-f41c0950328b">
change the sub label

Reply:
Replaced with a placeholder for the time being. Will replace the text once confirmed with product.